qubes-i3status

```diff
@@ -107,7 +107,9 @@ status_disk() {
     local free=''
     local size=''
     local usage=''
-    read size usage <<< $(qvm-pool -i $(qubes-prefs default-pool) | grep 'size\|usage' | sort | awk '{print $2}')
+    local qvmpool=$(qubes-prefs default-pool)
+    size=$(qvm-pool -i $qvmpool | grep 'size' | awk '{print $2}')
+    usage=$(qvm-pool -i $qvmpool | grep 'usage' | awk '{print $2}')
```

Comment:
This makes two calls to `qvm-pool` for each update, instead of one. Those aren't exactly the cheapest ones - better not duplicate them.

Reply:
Maybe something like this:

```bash
local -A pool
for line in $(qvm-pool -i $(qubes-prefs default-pool)); do
  read k v <<< "$line"
  pool[$k]=$v
done
```

Then access size (and usage) with `${pool["size"]}`?

Reply:
@cloneable thx, though this is not working for me, hence the two calls. Any suggestions welcome.

Reply:
Maybe just append a `tr '\n' ' '` to the end of the original line? Works for me at least.
```
read size usage <<< $(qvm-pool -i $(qubes-prefs default-pool) | grep 'size\|usage' | sort | awk '{print $2}' | tr '\n' ' ')
```

Reply:
Resolved in https://github.com/QubesOS/qubes-desktop-linux-i3-settings-qubes/pull/12/commits/78d9388b75e6be09438ebeb7ec7e69b304ca2f5e